# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RendererOGL.h
# Commit: f3b490c076d6
# Full Hash: f3b490c076d623d6ef3bcfd297a0522d9727477c
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-01-23 21:42:24
# Regressor Bug: 1602643
# File Overlap Count: 1
# Description:
#   Bug 1602643 - Disable WebRender compositor dinamically for async screenshot r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59658
# ==============================================================================

diff -r 0c5fb3a8a94d -r f3b490c076d6 gfx/webrender_bindings/RendererOGL.h
--- a/gfx/webrender_bindings/RendererOGL.h	Thu Jan 23 15:35:45 2020 +0200
+++ b/gfx/webrender_bindings/RendererOGL.h	Thu Jan 23 04:37:28 2020 +0000
@@ -109,6 +109,8 @@
 
   gl::GLContext* gl() const;
 
+  bool EnsureAsyncScreenshot();
+
  protected:
   RefPtr<RenderThread> mThread;
   UniquePtr<RenderCompositor> mCompositor;
@@ -117,6 +119,8 @@
   wr::WindowId mWindowId;
   TimeStamp mFrameStartTime;
 
+  bool mDisableNativeCompositor;
+
   RendererScreenshotGrabber mScreenshotGrabber;
 };
 